src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+
```

Comment:
[Don't begin class or method implementation with an empty line.](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-begin-class-or-method-implementation-with-an-empty-line)

Check all places

Reply:
i wrote about this, it's for checkstyle, cuz otherwise, project won't build, you can check it personally

Reply:
I checked locally and there are no errors.

Reply:
So you can stick to the rules and don't add empty line